src/wslsys.sh

```diff
@@ -3,10 +3,14 @@ version="28"
 help_short="wslsys (-h|-v|-S|-U|-b|-B|-fB|-R|-K|-P) -s"
 
 ## Windows 10 information
-branch=$(winps_exec "(Get-ItemProperty 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion').'BuildBranch'")
-build=$(winps_exec "(Get-ItemProperty 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion').'CurrentBuild'")
-full_build=$(winps_exec "(Get-ItemProperty 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion').'BuildLabEx'")
-installdate=$(winps_exec "(Get-ItemProperty 'HKLM:\Software\Microsoft\Windows NT\CurrentVersion').'InstallDate'")
+branch=`$(interop_prefix)c/Windows/System32/reg.exe query "HKLM\\Software\\Microsoft\\Windows NT\\CurrentVersion" /v BuildBranch | tail -n 2 | head -n 1`
```

Comment:
you should trim the `^M` at the very end here(using `sed -e 's|\r||g'`), and also use `$()` instead of ``

Reply:
Thanks (didn't notice the ^M that appeared in there; I was using Windows Terminal which apparently doesn't show anything wrong with that).

Reply:
well, you won't notice the ^M as it is hidden in the output, you have to use `cat -A` to see the character, as it is part of Windows' CRLF

Reply:
I see, thanks and thanks for taking care of this. I'll see if I notice something else that might need work. Any idea when this version reaches the repositories?

Reply:
Once I complete `wslusc` interactive mode, this version will reach the repositories.

Reply:
Alright, will see in my free time if I can help. Pretty nice idea, I could use these (the wslfetch banner could be of use, I'll see if I can help with the caching thing)